PatientMonitoringSystem/PatientMonitoringSystem/Views/ControlSystemView.cs

```diff
@@ -49,7 +51,7 @@ private void AddBedsideSystem(Guid bedsideSystemId)
 
 		public void ViewBedsideSystem(Guid bedsideSystemId)
 		{
-			new BedsideSystemView(bedsideSystemId).ShowDialog(this);
+			bed_gui_map[bedsideSystemId].ShowDialog(this);
```

Comment:
@harrytpstudent Is there a reason why we are caching the views now?

Reply:
This is because before we were creating new views every time we clicked the add view button.  This meant that new controllers were being created, as the controllers are created inside of the view object.  When the form was closed, we had no reference to the view and the controller was lost, so the timers were leaked.

They are cached now so when we click the view button we can get the correct reference to the form and also have the controller there still.